web/docs/auth/overview.md

```diff
@@ -429,65 +429,6 @@ If you use [Email](/docs/auth/email) authentication, the default validations are
 
 Note that `email`s are stored in a **case-insensitive** manner.
 
-### Customizing validations
```

Comment:
Removing docs section around using auth validation customisation since it no longer exists. 

I'd move the auth customization API to the DSL (like we have custom signup fields for example)